host_os.py

```diff
@@ -54,14 +53,15 @@
               "script: %s" % ", ".join(missing_packages))
         sys.exit(MISSING_PACKAGES_ERROR)
 
-    if os.getuid() is 0 and subcommand != 'set-env':
-        print("Please, do not run this command as root, run "
-              "host_os.py set-env --user <YOUR_USER_LOGIN> command in order to "
-              "properly setup user and directory for build scripts")
+    if os.getuid() is 0:
+        print("Please, do not run this command as root.")
```

Comment:
For this specific scenario, user must receive a visual feedback when fails to run host_os.py as root. Forcing the user to check the logs just to figure out what is wrong, is counterproductive for this case .